release/models/segment-routing/openconfig-segment-routing.yang

```diff
@@ -35,7 +36,13 @@ module openconfig-segment-routing {
       - SR SID advertisements - instantiated within the relevant IGP.
       - SR-specific counters - instantied within the relevant dataplane.";
 
-  oc-ext:openconfig-version "0.3.1";
+  oc-ext:openconfig-version "0.3.2";
```

Comment:
The minor release should be updated, ie: `0.4.0`

Reply:
Updated to 0.4.0